README.md

```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
 
-A command-line tool to manage cluster deployment with Kubernetes
+Additionaly, the schema of the file is versioned with the `apiVersion` field, so when new features are introduced you can switch to a newer version of the configuration file structure.
 
-Usage:
-  furyctl [command]
+To create a sample configuration file as a starting point use the following command:
 
-Available Commands:
-  bootstrap   Creates the required infrastructure to deploy a battle-tested Kubernetes cluster, mostly network components
-  cluster     Creates a battle-tested Kubernetes cluster
-  completion  Generate completion script
-  help        Help about any command
-  init        Initialize the minimum distribution configuration
-  vendor      Download dependencies specified in Furyfile.yml
-  version     Prints the client version information
+```console
+furyctl create config --version <KFD version> --kind "EKSCluster"
 ```
 
-## Download and manage KFD modules
+> 💡 **TIP**
+>
+> You can pass some additional flags, like the schema (API) version of the configuration file or a different configuration file name.
+>
+> See `furyctl create config --help` for more details.
 
-`furyctl` can be used as a package manager for the KFD.
-It provides a simple way to download all the desired modules of the KFD by reading a single `Furyfile`.
+Open the generated configuration file with your editor of choice and edit it according to your needs. You can follow the instructions included as comments in the file.
 
-The process requires the following steps:
+Once you have filled your configuration file, you can check that it's content is valid by running the following comand:
+
+```console
+furyctl validate config --config <path to your config file>
+```
+
+> **Note** the `--config` flag is optional, set it if your configuration file is not named `furyctl.yaml`
+
+#### 2. Create a cluster
+
+In the previous step, you have created and validated a configuration file that defines the Kubernetes cluster and its sorroundings, you can now proceed to actually creating the resources.
+
+furyctl has divided the cluster creation in three phases: `infrastructure`, `kubernetes` and `distribution`.
 
-1. Write a `Furyfile`
-2. Run `furyctl vendor` to download all the modules
-
-### 1. Write a Furyfile
-
-A `Furyfile` is a simple YAML formatted file that lists which modules (and versions) of the KFD you want to download.
-
-An example `Furyfile` is the following:
-
-```yaml
-# Here you can specify which versions of the modules to use
-versions:
-  networking: v1.7.0
-  monitoring: v1.13.0
-  logging: v1.9.1
-  ingress: v1.11.2
-  dr: v1.8.0
-  opa: v1.5.0
-
-# The bases are a sets of Kustomize bases to deploy Kubernetes components
-bases:
-  - name: networking/
-  - name: monitoring/
-  - name: logging/
-  - name: ingress/
-  - name: dr/
-  - name: opa/
+1. The first phase, `infrastructure`, creates all the prerequisites needed to be able to create a cluster. For example, the VPC and its networks.
```

Comment:
Let's add somewhere that this phase will also connect the client to the VPN

Reply:
Added a note right before the `create cluster` command example:
https://github.com/sighupio/furyctl/blob/1a9794e5e3d9f24fa44a59dfc3906d33a85a10c4/README.md?plain=1#L210